recipes-wpe/wpeframework/include/firmwarecontrol.inc

```diff
@@ -0,0 +1,38 @@
+FIRMWARECONTROL_PLATFORM ?= "none"
+WPE_FIRMWARECONTROL_EXTRAFLAGS += "${@bb.utils.contains('FIRMWARECONTROL_PLATFORM', 'rpi', '\
+     -DPLUGIN_FIRMWARECONTROL_PLATFORM_RPI=ON \
+', '', d)}"
```

Comment:
I would concatenate this to 2 liner like:
`WPE_FIRMWARECONTROL_EXTRAFLAGS += "${@bb.utils.contains('FIRMWARECONTROL_PLATFORM', 'rpi',
     '-DPLUGIN_FIRMWARECONTROL_PLATFORM_RPI=ON ', '', d)}"`